x-pack/plugins/cross_cluster_replication/public/app/services/auto_follow_pattern_validators.js

```diff
@@ -88,7 +88,25 @@ export const validateLeaderIndexPattern = (indexPattern) => {
     };
   }
 
-  return { message: null };
+  return null;
+};
+
+export const validateLeaderIndexPatterns = (indexPatterns) => {
+  if (!indexPatterns.length) {
+    return {
+      message: i18n.translate('xpack.crossClusterReplication.autoFollowPattern.leaderIndexPatternValidation.isEmpty', {
+        defaultMessage: 'At least one leader index pattern is required.',
+      })
+    };
+  }
+
+  let error = null;
```

Comment:
What do you think of adding a TODO comment here?

```js
// TODO: This is a bit inefficient because we're re-validating existing index patterns which have already
// been validated when the user submitted them in the combo box. In the future we can try to structure
// the code in a readable way that only performs validation as it's needed.
```